app/index.html

```diff
@@ -308,6 +303,15 @@ <h1 ng-bind="globalPage.title"></h1>
   </script>
   <!-- /build -->
 
-  <script type="text/javascript">_satellite.pageBottom();</script>
+  <!-- Adobe DTM -->
+  <script>
+    window.digitalData = {};
+  </script>
+  <!-- build:remove:dist -->
```

Comment:
Idk... Loading dynamically with jQuery works... It just doesn't feel like the right way to do that... If grunt can't do it, maybe we should move the ERT to webpack. I don't think it would take too much to just import a list of files and get the css working. We don't need to make it do anything fancy at first although we would need to add the translation tooling, assuming that has started being used.

Also that seems like it will delay the loading of DTM. From Adobe's DTM docs:

> Why does dynamic tag management have two parts?
> Dynamic tag management allows you to precisely time the loading of javascript and page content in your page. You can choose to run rules at the beginning or end of a page. This allows you to implement testing tools and other technologies, while retaining full control over tracking your pages.

Idk that Cru is doing anything with analytics that needs to distinguish between page top and bottom but idk what impact this will have on what's built into analytics.

Maybe this issue should have been addressed outside of this PR... It's my fault for bringing it up haha 😄  Or maybe just go with your jQuery solution and we'll hope to upgrade it soon.